### DISCUSSION TITLE
Orbat mapper in react

### DISCUSSION THREAD

**Discussion by Vbsasuke26** [orbat-mapper/orbat-mapper#214] [Q&A] [answered]
Hello Everyone.Is there any version of orbat mapper available in React instead of Vue?Thank you.

**Comment by kjellmf** [comment, marked as answer]
There is no React version of ORBAT mapper available and I don't have any plans for a rewrite.

**Reply by Vbsasuke26** [reply]
Thank you for replying. Just to know , is there any specific reason to use Vue ??

**Reply by kjellmf** [reply]
No, just a personal preference. React has more component libraries available, so it would probably be easier to write the user interface using React.
